$NetBSD: patch-Cargo.toml,v 1.41 2026/04/26 10:45:24 wiz Exp $

Depends on unpublished crates. Can't use cargo.mk to fetch it.

--- Cargo.toml.orig	2006-07-24 01:21:28.000000000 +0000
+++ Cargo.toml
@@ -125,7 +125,7 @@ lsp-server = { version = "0.7.6" }
 libcst = { version = "1.8.4", default-features = false }
 log = { version = "0.4.17" }
 lsp-server = { version = "0.7.6" }
-lsp-types = { git = "https://github.com/astral-sh/lsp-types.git", rev = "e15db0593f0ecbbd80599c3f5880e4bf5da1ca0c", features = [
+lsp-types = { path = "../lsp-types-e15db0593f0ecbbd80599c3f5880e4bf5da1ca0c", features = [
     "proposed",
 ] }
 matchit = { version = "0.9.0" }
